browser/module_test.go

```diff
@@ -0,0 +1,34 @@
+package browser
+
+import (
+	"testing"
+
+	"github.com/dop251/goja"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/xk6-browser/chromium"
+
+	k6common "go.k6.io/k6/js/common"
+	k6modulestest "go.k6.io/k6/js/modulestest"
+	k6metrics "go.k6.io/k6/metrics"
+)
+
+// TestModuleNew tests registering the module.
+// It doesn't test the module's remaining functionality as it is
+// already tested in the tests/ integration tests.
```

Comment:
Shall the `tests` then just import `browser` and test it ?

Reply:
preferably not in this PR but later ;)

Reply:
That makes sense, thanks :) However, I liked keeping this simple test next to the module creation. So that it would be easier to make changes to both code and tests when needed (instead of jumping to the other parts of the code base `tests/`.